oaknational/Oak-Web-Application
# LESQ-216 feat(quizzes): put quizzes inside new lesson item container

## Description

put quizzes inside lesson item container

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1792--oak-web-application.netlify.thenational.academy/beta/teachers/programmes/french-primary-ks2/units/je-decris-un-monstre-8348/lessons/describing-parts-of-the-head-and-face-with-colours-and-sizes-6gv3jd
2. Click on starter and exit quiz
3. You should see new styled quizzes

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
